plugins/auth-supertokens/package.json

```diff
@@ -0,0 +1,60 @@
+{
+  "name": "@amplication/plugin-auth-supertokens",
+  "version": "0.0.1",
+  "description": "Use supertokens for authentication in the service generated by Amplication",
+  "main": "dist/index.js",
+  "scripts": {
+    "prepublishOnly": "npm run build",
+    "dev": "webpack --watch",
+    "build": "webpack",
+    "prebuild": "rimraf dist",
+    "test": "jest",
+    "format:write": "prettier --write \"**/*.{js,ts}\""
+  },
+  "author": "Demilade Sonuga",
+  "license": "Apache-2.0",
+  "devDependencies": {
+    "@amplication/code-gen-types": "^2.0.12",
+    "@amplication/code-gen-utils": "^0.0.5",
```

Comment:
@d-sonuga  do you mind updating these dependencies to the latest? 
plugin-template should be up-to-date

Reply:
@overbit , I've updated the dependencies.